test/scenarios/cloudnative/istio/istio_test.go

```diff
@@ -26,7 +26,7 @@ func TestMain(m *testing.M) {
 	testEnvironment.Run(m)
 }
 
-func TestCloudNative(t *testing.T) {
-	testEnvironment.Test(t, basic.Install(t, true))
+func TestIstioCodeModules(t *testing.T) {
```

Comment:
Not that true as you test both the default case and the codemodules case.

Also the `codemodules` case is kinda not super clear, as its the `codeModulesImage` case 🙃
`codemodule` == `apponly agent`, so cloudnative always has codemodules, but the case we are talking about uses the `codeModulesImage` field that downloads the codemodule in image instead of a zip.

is this confusing? yes 😬

Reply:
@0sewa0 it's not clear, do you want to rename it to `TestIstioCodeModulesImages`?

Reply:
2 things: 
1. The [CodeModules](https://github.com/Dynatrace/dynatrace-operator/pull/2104/files#diff-cccfe1f91b1ae916741c6f93c5ae4ca3288326700703a7f2e646c796dd6d06a4R25) test should probably be named `CodeModulesImage`, so its more explicit
2. The test `TestIstioCodeModules` runs both the (CloudNative)`Default` and `CodeModules` tests, so `TestIstioCodeModules` is not very fitting.
     - We can rename it to `TestIstio` or `TestIstioIntegration`
     - Or we can split them into separate function one for `Default` and one for `CodeModules`, but this would require some changes to the `make` targets
     
Sidenote: These istio e2e tests are rather ineffective(see my [PR](https://github.com/Dynatrace/dynatrace-operator/pull/2105) on how little they check), and I will create a follow up for them to be made nicer, so I don't reeeally care that much about them right now 😅

Reply:
fixed in e908f741